dspace/config/modules/publication-lookup.cfg

```diff
@@ -1 +1 @@
-publication.url = ${publication.url}
\ No newline at end of file
+publication-lookup.publication.url = ${publication.url}
```

Comment:
@tdonohue Sorry for the delay in replying.

The reason for this property is the ability to have it configurable which "importSource" to use (and thus base the retrieved results on).
With the `*` configured all configured "ImportService" are used. (Upon searching records, all results are concatenated) But if we want to focus on one specific Service, it's possible to specify a publication.url (which is the base-address that is configured in spring-dspace-addon-import-services.xml), and only the results of that ImportService are retrieved.

For example, here we have only 1 ImportService subclass specified, so it doesn't matter if the actual base-address is filled in, or simply the asterisk `*`. Since it will return the same results.
https://github.com/DSpace/DSpace/blob/master/dspace-api/src/main/resources/spring/spring-dspace-addon-import-services.xml#L27

But if a second config is added, it could be useful to have an option to only retrieve from a certain source, or from all of them.

I do however, agree that the default could be set to the `*`, but still retaining the possibility to specify the ImportServices for aforementioned reasons.